modules/farm_fd2_school/07-FD1-Tutorial-Starter/index.html

```diff
@@ -54,17 +41,17 @@ <h1>07 FD1 Tutorial</h1>
           };
         },
         methods: {
-          fetchUsers() {
-            this.userList = [
-              { id: 1, name: 'Leanne Graham' },
-              { id: 2, name: 'Ervin Howell' },
-            ];
-          },
-          clearUsers() {
-            this.userList = [];
-            this.selectedUser = null;
+          async fetchUsers() {
+            const usersResponse = await fetch(
+              'https://jsonplaceholder.typicode.com/users'
+            );
+            const users = await usersResponse.json();
+            this.userList = users;
           },
         },
+        created() {
```

Comment:
The `created` lifecycle hook calls the `fetchUsers` method which is `async` so the `created` lifecycle hook should also be `async`.